src/halmos/sevm.py

```diff
@@ -253,9 +253,10 @@ def wstore_bytes(
         raise ValueError(size, arr)
     wextend(mem, loc, size)
     for i in range(size):
-        if not eq(arr[i].sort(), BitVecSort8):
+        arr_i_bv = arr[i] if is_bv(arr[i]) else con(arr[i], 8)
```

Comment:
questions:  when arr[i] is not bv:
- do we want to store them as bv, rather than keeping it int?
- do we need to check if arr[i] < 256, or is it guaranteed?

Reply:
nice catch, can you check again with [7c5c364](https://github.com/a16z/halmos/pull/239/commits/7c5c3647f8c4a99451a1852ce607977d9098d5fe)?